deploy.sh

```diff
@@ -7,6 +7,7 @@ if [ "$#" -ne 1 ]; then
 	exit 1
 elif [ "$1" == "build" ]; then
     ./deploy.sh clean
+    ./deploy.sh gtest-setup
```

Comment:
We actually don't want to run gtest-setup at build. We would ideally be building several times but only want to run gtest-setup once to install the packages.